python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -747,26 +734,26 @@ def __str__(self) -> str:
         return f"{self.left.operand__str__()} - {self.right.operand__str__()}"
 
 
-@whitelist_for_serdes
-class SinksAssetSelection(AssetSelection):
+class ChildBasedAssetSelection(AssetSelection):
```

Comment:
Brief docblock would be helpful

Reply:
"Chained" also possible naming here if we are considering that.  "Child" slightly odd because frequently (usually?) the child is actually the superset of the parent and because the "child" verbiage not used in the operands where it is actually more natural.

Child would become inner I think.

However do not feel strongly.